src/domain/synthetics/stats/useVolumeInfo.ts

```diff
@@ -0,0 +1,78 @@
+import { gql } from "@apollo/client";
+import { BigNumber } from "ethers";
+import { getSyntheticsGraphClient } from "lib/subgraph";
+import useSWR from "swr";
+
+type VolumeInfo = {
+  dailyVolume: BigNumber;
+  totalVolume: BigNumber;
+};
+
+const query = gql`
+  query volumeInfo($lastTimestamp: Int!) {
+    hourlyVolumeInfos: volumeInfos(where: { id_gte: $lastTimestamp, period: "1h" }) {
+      volumeUsd
+      id
+    }
+    totalVolumeInfos: volumeInfos(where: { period: "total" }) {
+      volumeUsd
+      period
+      id
+    }
+  }
+`;
+
+export default function useVolumeInfo(chains: number[]) {
+  const lastPeriodFor24Hours = Math.floor(Date.now() / 1000 / 3600) * 3600 - 60 * 60 * 24;
```

Comment:
the time here depends on hook first render time, then if swr key is not changed, it will not update the fetcher function and will refer to the initial time value

we need to move this into fetcher function